README.md

```diff
@@ -268,6 +268,17 @@ arrayType.push('d');
 console.log(arrayType.toValue()); // ['a', 'b', 'c', 'd']
 ```
 
+##### find
+
+The `find` method traverses the element tree and returns an `ArrayType` of all elements that match the conditional function given.
+
+```javascript
+var arrayType = new minim.ArrayType(['a', [1, 2], 'b', 3]);
```

Comment:
BTW, I think we should not call these ArrayTypes but rather ArrayElements. In that way we are aligning our code with the refract domain. I can address this in the forthcoming PR where I implement the factory and break up the files.